### DISCUSSION TITLE
Travel menu appears at the start of "flipped" levels

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13809] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

If you travel from right to left on the map, the levels are "flipped", sometimes resulting in the submarine spawning on top of the starting point.

**1. This will open the map menu (happens when you reach a level exitpoint), which can be confusing/annoying**

**2. If the menu opens, this will cause a bug when reaching the destination: sometimes you won't be able to travel to the location and will have to pick another one.** 

Example:
I am travelling to this location, which can be entered by getting near (exitpoint)
However, the game will not let me enter the location if the map has been opened on round start

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/b6042ec3-f6c8-4cb1-9ccf-d147bf01ac40)

### Reproduction steps

Download 
[#DELocalHuskModIntegrated.zip](https://github.com/FakeFishGames/Barotrauma/files/15079174/DELocalHuskModIntegrated.zip)

1. Start a campaign
2. togglecampaignteleport 
3. Teleport to an empty node near a lair
4. Map should open on start of level
5. Travel to the end of the level
6. You can't go to your destination

### Bug prevalence

Just once

### Single player or multiplayer?

Other

### Version

v1.4.3.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
> 2. If the menu opens, this will cause a bug when reaching the destination: sometimes you won't be able to travel to the location and will have to pick another one.

You can also just close the map, you're not forced to choose a destination. But yeah, this is definitely confusing and should be addressed.

**Reply by ubertpendragon** [reply]
No there's an actual bug if you're using Exitpoint; you can't travel to your destination, probably related to https://github.com/FakeFishGames/Barotrauma/discussions/13636


We have those locations called "lairs", which are flooded, cave-like outposts with monsters in it. 

To enter an outpost level, you usually need to dock. This can be circumvented by adding an Exitpoint spawnpoint

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/989b4d19-f773-4322-a1f3-f9b5effd9adc)


The big issue here was that, if the map travel menu opened on level start, it was impossible to travel to the outpost (as seen in screenshot). 
Your travel options were as if you went back to the start of the level and triggered travel. Maybe because if there are multiple exitpoints in a level, they all share the same travel rules.


Ultimately not even opening the map menu upon reaching those manually placed exitpoints would be an option. 
The anomaly in the end entrance level has an exit point, but it seems hardcoded how it teleports you to the next level

**Comment by Regalis11** [comment]
Created a ticket about this: https://github.com/FakeFishGames/Barotrauma/issues/13862

**Comment by Rokvach** [comment]
Tested working correctly. Closing.
